python/packages/autogen-agentchat/src/autogen_agentchat/agents/_user_proxy_agent.py

```diff
@@ -51,7 +59,7 @@ def __init__(
     ) -> None:
         """Initialize the UserProxyAgent."""
         super().__init__(name=name, description=description)
-        self.input_func = input_func or input
+        self.input_func = input_func or cancellable_input
```

Comment:
Can you please add a test that showed a sync input function is cancellable

Reply:
i dont think a sync input fn is cancellable